operation/fhir-operation-bulkdata/src/main/java/com/ibm/fhir/operation/bulkdata/util/CommonUtil.java

```diff
@@ -47,10 +48,16 @@ public void checkAllowed(FHIROperationContext operationContext, boolean isImport
             String source = adapter.getStorageProvider();
             String outcome = adapter.getStorageProviderOutcomes();
 
-            boolean s = ConfigurationFactory.getInstance().hasStorageProvider(source);
-            boolean o = ConfigurationFactory.getInstance().hasStorageProvider(outcome);
-            if (!s || !o) {
-                throw FHIROperationUtil.buildExceptionWithIssue("The storage provider for outcome [" + s + "] or source [" + o + "] passed is not configured properly" , IssueType.EXCEPTION);
+            if (!ConfigurationFactory.getInstance().hasStorageProvider(source)
+                    || !ConfigurationFactory.getInstance().hasStorageProvider(outcome)) {
```

Comment:
it seems ok to be missing an outcome storageProvider if outcomes are disabled.  maybe split this into two different blocks and only emit an error on the outcome one if we're emitting outcomes?